main.go

```diff
@@ -30,10 +31,15 @@ func main() {
 }
 
 func main1() int {
-	var err error
+	gunkConfig, err := config.Load("")
```

Comment:
Should the config store the dir (which is `""` in this case)? If so maybe it should also store the patterns?

Reply:
I think the config should keep the path it was found at - e.g. `/home/mike/src/brankas-proto/.gunkconfig`. It can be useful to report errors at a later time, such as `found unknown generator "foo" in .../foo/bar/.gunkconfig`.

I don't think keeping the search start directory as part of the config struct will be helpful, unless you have a specific use case for it.